lenses/dovecot.aug

```diff
@@ -114,7 +114,7 @@ let block_newlines (entry:lens) (comment:lens) =
 Map block enclosed in brackets recursively.
 Block may be indented and have optional argument.
 Block body may have entries, comments, empty lines, and nested blocks recursively. *)
-let rec block = [ indent . key block_names . (Sep.space . Quote.do_dquote_opt (store /[\/A-Za-z0-9_-]+/))? . block_newlines (entry|block|mailbox) comment . eol ]
+let rec block = [ indent . key block_names . (Sep.space . Quote.do_dquote_opt (store /!?[\/A-Za-z0-9_-]+/))? . block_newlines (entry|block|mailbox) comment . eol ]
```

Comment:
Is the `!` really allowed inside the quotes?

Reply:
Sorry - I'd missed this comment.

Yes it is - either inside or outside has the same behaviour. I'm sorry I'm not familiar with the dovecot config parser code enough to examine the code - but testing seems to indicate that it's OK.

I don't know how we could permit "inside or outside" here.

Reply:
OK. Could you provide some examples of valid syntax in the test file?